### PR TITLE
added 3 projects to data and repositories

### PR DESCRIPTION
**Pull request by leokim-l**
Added AutoMAxo, phenopacket2prompt, and pheval.llm in the tools field and repositories field. What remains to be done is to fill in the info beyond name and id in the data field's lines

### CODE REVIEW

File: src/data/resources.yaml [hunk 1]
```diff
@@ -968,4 +1007,9 @@ repositories:
     name: Fenominal
   - id: https://github.com/monarch-initiative/monarch-ontology-dashboard
     name: Monarch Ontology Dashboard
-
+  - id: https://github.com/monarch-initiative/pheval.llm
+    name: pheval.llm
+  - id: https://github.com/monarch-initiative/
```
Comment (matentzn) [inline src/data/resources.yaml:1012]: something is missing here?

File: src/data/resources.yaml [hunk 2]
```diff
@@ -856,6 +856,45 @@ tools:
...
         orcid: https://orcid.org/0000-0003-3311-7320
         github: "@kevinschaper"
+  - id: https://github.com/monarch-initiative/pheval.llm
+    name: pheval.llm
+    category: Benchmarking
+    description: LLM Benchmarking in differential diagnostics.
+    repository: https://github.com/monarch-initiative/pheval.llm
+    documentation: https://monarch-initiative.github.io/pheval.llm
+    monarch_role: Support
+    monarch_contribution: Co-Lead
```
Comment (matentzn) [inline src/data/resources.yaml:866]: Why Co-Lead?

**Review by matentzn — COMMENTED** [review]
Very nice two q's all makes sense

File: src/data/resources.yaml (hunk 1 above)
Comment (leokim-l) [reply src/data/resources.yaml]: Fixed it

File: src/data/resources.yaml (hunk 2 above)
Comment (leokim-l) [reply src/data/resources.yaml:866]: You decide as per our discussion

**Comment by leokim-l** [comment]
If Enock is fine with AutoMAxO feel free to merge

**Pull request merged**